RDFstar-demo.md

```diff
@@ -1,66 +1,76 @@
 ## Complex modelling
 
-Below some examples of how to model complex concepts in RDF
+Below some examples of how to model complex relationships in RDF
 
 #### 1. Standard reification
 
 ```
-:man :hasSpouse :woman .
+:bob :hasSpouse :alice .
 :id1 rdf:type rdf:Statement ;
-    rdf:subject :man ;
+    rdf:subject :bob ;
     rdf:predicate :hasSpouse ;
-    rdf:object :woman ;
+    rdf:object :alice ;
     :startDate "2020-02-11"^^xsd:date .
 ```
 
 #### 2. N-ary relations
 
 ```
-:Marriage1 rdf:type :Marriage ;
-    :partner1 :man ;
-    :partner2 :woman ;
+:marriage1 rdf:type :marriage ;
+    :partner1 :bob ;
+    :partner2 :alice ;
     :startDate "2020-02-11"^^xsd:date .
 ```
 
-#### 3. Named Graphs
+#### 3. Named Graphs (TriG notation)
 
 ```
-:man :hasSpouse :woman :statementId#1 .
-:statementId#1 :startDate "2020-02-11"^^xsd:date :metadata .
+@prefix : <http://www.example.org/> .
+:statementId#1 { :bob :hasSpouse :alice }
+{ :statementId#1 :startDate "2020-02-11"^^xsd:date :metadata . }
```

Comment:
That is still invalid TriG.
```suggestion
:metadata { :statementId#1 :startDate "2020-02-11"^^xsd:date }
```

Reply:
Ah right. I wanted to make it the default graph not yet another graph with a name but forgot to delete ":metadata". Maybe remove ":metadata"?

Reply:
yes, I am fine with removing :metadata as well... in which case the brackets are not even required.
```suggestion
:statementId#1 :startDate "2020-02-11"^^xsd:date .
```